libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c

```diff
@@ -112,6 +112,9 @@ type. */
 	#endif
 #endif
 
+/* Define the ASCII value of '.' (Period/Full-stop). */
+#define ASCII_BASELINE_DOT						46UL
```

Comment:
As `ASCII_BASELINE_DOT` will be cast to `uint8_t`, i think the 'L' suffix is not necessary, `46U` is enough.

Reply:
Done, thanks!